microsoft-edge/webview2/how-to/print.md

```diff
@@ -0,0 +1,793 @@
+    DisplayPdfDataInPrintDialog(stream);
+
+    MessageBox.Show(this, "Printing " + title + " document to PDF Stream " + 
+                ((stream != null) ? "succedded" : "failed"), "Print To PDF Stream");
+  }
+  catch(Exception exception)
+  {
+    MessageBox.Show(this, "Printing to PDF Stream failed: " + exception.Message, "Print to PDF Stream");
+  }
+}
+
+// Function to display current web page pdf data in a custom print preview dialog.
+void DisplayPdfDataInPrintDialog(Stream pdfData)
+{
+  // You can display the printable pdf data in a custom print preview dialog to the end user.
+}
+```
+
+**APIs:**
+
+* [CoreWebView2.PrintToPdfStreamAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfstreamasync#microsoft-web-webview2-core-corewebview2-printtopdfstreamasync)
+
+
+<!-- ------------------------------ -->
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+Pending.<!-- todo -->
+
+
+<!-- ------------------------------ -->
+
+
+##### [Win32/C++](#tab/win32cpp)
+
+```cpp
+static void DisplayPdfDataInPrintDialog(IStream* pdfData)
+{
+  // You can display the printable pdf data in a custom print preview dialog to the end user.
+}
+
+bool AppWindow::PrintToPdfStream()
+{
+  wil::com_ptr<ICoreWebView2_16> webView2_16;
+  CHECK_FAILURE(m_webView->QueryInterface(IID_PPV_ARGS(&webView2_16)));
+
+  wil::unique_cotaskmem_string title;
+  CHECK_FAILURE(m_webView->get_DocumentTitle(&title));
+
+  // Passing nullptr for `ICoreWebView2PrintSettings` results in default print settings used.
+  CHECK_FAILURE(
+      webView2_16->PrintToPdfStream(
+          nullptr,
+          Callback<ICoreWebView2PrintToPdfStreamCompletedHandler>(
+              [title = std::move(title), this](HRESULT errorCode, IStream* pdfData) -> HRESULT
+              {
+                DisplayPdfDataInPrintDialog(pdfData);
+                std::wstring message = L"Printing " + std::wstring(title.get()) +
+                                       L" document to PDF Stream " +
+                                       ((errorCode == S_OK && pdfData != nullptr) ? L"succedded" : L"failed");
+
+                AsyncMessageBox(message, L"Print to PDF Stream");
+
+                return S_OK;
+              })
+              .Get()));
+  return true;
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_16::PrintToPdfStream method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#printtopdfstream)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## See also
+
+* [Printing](../concepts/overview-features-apis.md#printing) in _Overview of WebView2 features and APIs_.
+* [Get started with WebView2](../get-started/get-started.md)
+* [WebView2Samples repo](https://github.com/MicrosoftEdge/WebView2Samples) - a comprehensive example of WebView2 capabilities.
+* [WebView2 API reference](/dotnet/api/microsoft.web.webview2.wpf.webview2)
+* [See also](../index.md#see-also) in _Introduction to Microsoft Edge WebView2_.
```

Comment:
```suggestion
* [Introduction to Microsoft Edge WebView2](../index.md)
```
I need to do a PR to get rid of that design of linking from several articles' See Also sections to the See Also section of the topmost article.